exporter/lokiexporter/exporter.go

```diff
@@ -227,14 +227,50 @@ func (l *lokiExporter) convertAttributesToLabels(attributes pdata.AttributeMap,
 	return ls
 }
 
-func convertLogBodyToEntry(lr pdata.LogRecord, res pdata.Resource) (*logproto.Entry, error) {
+func (l *lokiExporter) convertLogBodyToEntry(lr pdata.LogRecord, res pdata.Resource) (*logproto.Entry, error) {
+	body := lr.Body().StringVal()
+
+	if len(lr.Name()) > 0 {
+		body = fmt.Sprintf("name=%s %s", lr.Name(), body)
```

Comment:
While I'm not deeply familiar with its performance in Go in most languages repeated String generation like this is very expensive.

How about using a `strings.Builder`?

https://pkg.go.dev/strings#Builder

Reply:
Done!